src/components/NewWorkspaceModal.js

```diff
@@ -154,13 +154,20 @@ const NewWorkspaceModal = Utils.withDisplayName('NewWorkspaceModal', ({
       ])]),
       h(IdContainer, [id => h(Fragment, [
         h(FormLabel, { htmlFor: id, required: true }, ['Billing project']),
+        div({ style: { margin: '0.5rem 0' } }, ['A billing project must have a billing account which Terra has access to in order to be used.']),
```

Comment:
I like the idea behind this message. I'm not a huge fan of the UX though, since this text is a bit separated from the disabled billing projects. Could we either:
* show this message on each disabled billing project (maybe with an info icon and a tooltip?) 
* or not display the disabled billing projects in the list.

Reply:
I thought this would affect a lot of users since I see it affecting 9500 projects on dev. I took a look on prod, and this only affects 69 projects (most of which are firecloud projects), so I do think this message adds visual clutter for most users.